internal/service/connect/wait.go

```diff
@@ -20,6 +20,9 @@ const (
 
 	connectHoursOfOperationCreatedTimeout = 5 * time.Minute
```

Comment:
I see these weren't implemented either in that resource. 

I see that Connect Instance does implement the waiter, but doesn't honor the user override from the config:
https://github.com/hashicorp/terraform-provider-aws/blob/f445f5e1980a8b4a6581b2964b8ba70b80330128/internal/service/connect/instance.go#L139

Reply:
Resolved in bc0343b81d70556e78c6399bf73e75b1751cf96f by removing timeouts